### DISCUSSION TITLE
Webhook saying invalid SSL

### DISCUSSION THREAD

**Discussion by jamesputty95** [postalserver/postal#1733] [Help with using Postal]
Hi there im using a webhook with mailwizz to process bounced emails and the response body says invalid SSL for all messages. Currently im using an IP pool with 3 separate subdomains under 3 IPs. No bounces are being processed by mailwizz and I believe it's because of this SSL issue.

postal.domain.com
postal1.domain.com
postal2.domain.com

All have a valid SSL certificate

The domain mailwizz is hosted on also has an SSL certificate.

How can I resolve this issue?

<img width="655" alt="History - Webhooks - greenpixie co IP 2 - Additional IP - Postal2 - Postal 2021-11-24 15-37-02" src="https://user-images.githubusercontent.com/81581752/143268744-b1e7c981-9319-41dd-adae-1151edcf814d.png">

**Comment by willpower232** [comment]
Are the certificates all provided by Lets Encrypt? Can you share your `postal status` to confirm if anything else is wrong?

**Comment by jamesputty95** [comment]
Hi there Will, thanks for the response. 

1. All subdomains on postal as well as my mailwizz server are all encrypted with Let's Encrypt


2
[postal status.docx](https://github.com/postalserver/postal/files/7604149/postal.status.docx)

**Reply by willpower232** [reply]
Your `postal status` confirms you are using Postal v1 which means you likely need to do software updates on the server running Postal. Lets Encrypt expired one of the chain certificates recently so older server software needs an update.

**Reply by jamesputty95** [reply]
Ah awesome I had no idea. Don't suppose you can point me towards some documentation for how to run the necessary software upgrades?

**Reply by jamesputty95** [reply]
nevermind i found it https://docs.postalserver.io/install/upgrade-to-v2/

**Reply by willpower232** [reply]
Glad you found it, I was meaning that you would need to do some kind of `apt-get update` or whatever is correct for your server. Upgrading to Postal v2 should resolve it as well but the server update would be faster if you don't have the time right now for a full upgrade 😅

**Reply by jamesputty95** [reply]
Hey apologies for the delay. Just did a sudo apt-get upgrade; sudo apt-get update and webhooks are working again. Thanks for the assistance
